src/pages/workspaces/workspace/Workflows.js

```diff
@@ -189,20 +198,68 @@ const FindWorkflowModal = ajaxCaller(class FindWorkflowModal extends Component {
 
     const { synopsis, managers, documentation } = selectedWorkflowDetails || {}
 
+    const errors = validate({ workflowRename }, {
+      workflowRename: {
```

Comment:
Validation constraints matched to those of naming a workflow when duplicating or copying.